### ISSUE TITLE
alignment of top logo and nav

### ISSUE THREAD

**Issue by nightfury-crypto** [devarshishimpi/Stone-CSS-Website#46]
Hey! I can do that please assign this to me.

`Before --`

![image](https://user-images.githubusercontent.com/65121678/192124714-aa93827f-7d1e-461b-842e-b859e2d63729.png)

`after --`
![image](https://user-images.githubusercontent.com/65121678/192124726-546ea1ca-5f81-49f9-b2ee-c49967e8928f.png)

**Comment by devarshishimpi** [comment]
Hi, Sure, Also Could You Share the Mobile Preview?

**Comment by nightfury-crypto** [comment]
> Hi, Sure, Also Could You Share the Mobile Preview?

I will create and show it to you.

**Comment by devarshishimpi** [comment]
> > Hi, Sure, Also Could You Share the Mobile Preview?
> 
> I will create and show it to you.

Sure.

**Comment by nightfury-crypto** [comment]
@devarshishimpi 

`desktop view`

![image](https://user-images.githubusercontent.com/65121678/192151735-fa21722e-b7e5-47f8-b025-0d88f471cdb2.png)

`mobile view`

![image](https://user-images.githubusercontent.com/65121678/192151698-a29370f2-f8d9-4ceb-9666-3b3b90f73a99.png)

this is ok?

**Comment by devarshishimpi** [comment]
Yes works, Do check it with the rest of the devices and sizes as well.

**Comment by nightfury-crypto** [comment]
@devarshishimpi 

I'm done with the header part but for now i'm commiting only index.html file if you think that's good I'll implement header to rest pages. ok?

**Comment by devarshishimpi** [comment]
> @devarshishimpi
> 
> I'm done with the header part but for now i'm commiting only index.html file if you think that's good I'll implement header to rest pages. ok?

Works.

**Comment by nightfury-crypto** [comment]
@devarshishimpi is there anything left related to this issue?

**Comment by devarshishimpi** [comment]
> @devarshishimpi is there anything left related to this issue?

Nope, It's completed, and Closed.
